openlibrary/catalog/marc/tests/test_data/bin_input/880_alternate_script.mrc

```diff
@@ -0,0 +1 @@
+01363cam a2200409Ia 4500001001300000003000600013005001700019008004100036040003000077066000700107020003000114020002700144035002100171041001300192043001200205092001500217090002900232100002600261240002200287245007700309250002200386260005700408300002500465600003800490610003500528650002500563651006200588700002300650880008100673880002400754880005200778880002300830987002400853029002200877994001200899948004200911ocn613515810OCoLC20130718194202.0100512s2010    cc            000 1 chi d  aCE@bengcCE@dOCLCQdHNK  c$1  a9787508617725 :cRMB29.00  a750861772X :cRMB29.00  a(OCoLC)6135158101 achiheng  an-us-ca0 a813.54222  aPS3562.Y4483bO6712 20101 aLyons, Daniel,d1960-10aOption$.lChinese106880-01aQiaobusi de mi mi ri ji /cDanni'er Lai'angsi zhu ; Liu Ning yi.  6880-02aDi 1 ban.  6880-03aBeijing Shi :bZhong xin chu ban she,c2010.  axi, 274 p. ;c22 cm.10aJobs, Steve,d1955-2011vFiction.20aApple Computer, Inc.vFiction. 0aExecutivesvFiction. 0aSanta Clara Valley (Santa Clara County, Calif.)vFiction.1 6880-04aLiu, Ning.106245-01/$1a乔布斯的秘密日记 /c丹尼尔・莱昂斯著 ; 刘宁译.  6250-02/$1a第1版.  6260-03/$1a北京市 :b中信出版社,c2010.1 6700-04/$1a刘宁.  aPINYINbCN-shCEIdc1 aAU@b000046817183  aZ0bPMR  hNO HOLDINGS IN PMR - 8 OTHER HOLDINGS
```

Comment:
```suggestion
01363cam a2200409Ia 4500001001300000003000600013005001700019008004100036040003000077066000700107020003000114020002700144035002100171041001300192043001200205092001500217090002900232100002600261240002200287245007700309250002200386260005700408300002500465600003800490610003500528650002500563651006200588700002300650880008100673880002400754880005200778880002300830987002400853029002200877994001200899948004200911ocn613515810OCoLC20130718194202.0100512s2010    cc            000 1 chi d  aCE@bengcCE@dOCLCQdHNK  c$1  a9787508617725 :cRMB29.00  a750861772X :cRMB29.00  a(OCoLC)6135158101 achiheng  an-us-ca0 a813.54222  aPS3562.Y4483bO6712 20101 aLyons, Daniel,d1960-10aOption$.lChinese106880-01aQiaobusi de mi mi ri ji /cDanni'er Lai'angsi zhu ; Liu Ning yi.  6880-02aDi 1 ban.  6880-03aBeijing Shi :bZhong xin chu ban she,c2010.  axi, 274 p. ;c22 cm.10aJobs, Steve,d1955-2011vFiction.20aApple Computer, Inc.vFiction. 0aExecutivesvFiction. 0aSanta Clara Valley (Santa Clara County, Calif.)vFiction.1 6880-04aLiu, Ning.106245-01/$1a乔布斯的秘密日记 /c丹尼尔・莱昂斯著 ; 刘宁译.  6250-02/$1a第1版.  6260-03/$1a北京市 :b中信出版社,c2010.1 6700-04/$1a刘宁.  aPINYINbCN-shCEIdc1 aAU@b000046817183  aZ0bPMR  hNO HOLDINGS IN PMR - 8 OTHER HOLDINGS

```

Reply:
@cclauss MARC is a binary format, I have (hopefully) switched off the git + github attempts to treat these as text files in diffs with this PR: #7662

Adding a newline at the end breaks the encoded length, and causes errors when trying to parse it. MARC records must end with a 0x1D (record terminator)

Reply:
I have rebased against the new .gitattributes -- command line git recognises the file as binary now with `git diff`, but I still see the warning mark in the github interface. I'm not sure if it's some kind of caching or whether the branch takes time to update (I know some of the Linguist analysis takes time to appear after changes are made).

I think we've done all we can do with correct `.gitattributes` -- I hope the github UI view comes right eventually.

Reply:
Just ignore it then.